rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/health/10_basic.yml

```diff
@@ -0,0 +1,14 @@
+---
+"cluster health basic test":
+  - do:
+      health: {}
+
+  - is_true:   cluster_name
+  - is_false:  timed_out
+  - match:     { status: "green" }
+  - match:     { impacts: [] }
+  - match:     { components.controller.status: "GREEN" }
+  - match:     { components.controller.indicators.0.status: "GREEN" }
+  - match:     { components.controller.indicators.0.name: "instance does not have master" }
```

Comment:
It was quiet confusing for me to see `GREEN` and then immediately `instance does not have master`.
I would suggest to avoid negations and reword to `Instance has a master`

Reply:
Yeah, it should be `health coordinating instance has a master node`